### PR TITLE
disabled the releases and access requests tab on model page if no sch…

### PR DESCRIPTION
**Pull request by ARADDCC002**
…ema selected

### CODE REVIEW

File: frontend/src/common/PageWithTabs.tsx [hunk 1]
```diff
@@ -90,10 +91,15 @@ export default function PageWithTabs({
           return (
             <Tab
               key={tab.title}
-              label={tab.title}
               disabled={tab.disabled}
               value={tab.path}
               data-test={`${tab.path}Tab`}
+              style={{ pointerEvents: 'auto' }}
```
Comment (ARADDCC012) [inline frontend/src/common/PageWithTabs.tsx:97]: Can we use `sx` here?

**Review by ARADDCC012 — CHANGES_REQUESTED** [review]

File: frontend/src/common/PageWithTabs.tsx [hunk 2]
```diff
@@ -90,10 +91,15 @@ export default function PageWithTabs({
           return (
             <Tab
               key={tab.title}
-              label={tab.title}
               disabled={tab.disabled}
               value={tab.path}
               data-test={`${tab.path}Tab`}
+              sx={{ pointerEvents: 'auto' }}
```
Comment (ARADDCC012) [inline frontend/src/common/PageWithTabs.tsx:97]: Is this needed? I believe the default behaviour of `Tab` should handle this for us.

**Review by ARADDCC012 — CHANGES_REQUESTED** [review]

File: frontend/src/model/beta/ModelImages.tsx [hunk 3]
```diff
@@ -51,12 +51,7 @@ export default function ModelImages({ model }: AccessRequestsProps) {
       <Container sx={{ my: 2 }}>
         <Stack spacing={4}>
           <Box sx={{ textAlign: 'right' }}>
-            <Button
-              variant='outlined'
-              onClick={() => setOpenUploadImageDialog(true)}
-              disabled={!model.card}
```
[the remark this answers is not in the document]
Comment (ARADDCC002) [reply frontend/src/model/beta/ModelImages.tsx:57]: Nah we discussed this and we reckon that Images can be added before a schema is set

File: frontend/src/common/PageWithTabs.tsx [hunk 4]
```diff
@@ -12,6 +12,7 @@ export interface PageTab {
   view: ReactElement
   disabled?: boolean
   datatest?: string
+  disabledText?: string
```
Comment (ARADDCC012) [inline frontend/src/common/PageWithTabs.tsx:15]: I don't think this is being used is it?

**Review by ARADDCC012 — APPROVED** [review]

**Pull request merged**